src/synapse_tools/generate_container_ip_map.py

```
@@ -63,7 +63,11 @@ def extract_taskid_and_ip_mesos(
 def extract_taskid_and_ip_k8s() -> Iterable[Tuple[str, str]]:
     service_ips_and_ids = []
 
-    node_info = requests.get("http://169.254.255.254:10255/pods/").json()
+    print('querying kubelet for pods')
+    response = requests.get("http://169.254.255.254:10255/pods/", timeout=10)
```

Not blocking - Since we run this generator every 5 seconds, may be good to fail fast here, so something < 5s? (may be 2 or 3?)

---

This is based on assumption that we want the file updated every 5s and not be stale for longer periods.

---

agreed - reduced timeout to 2 seconds